### DISCUSSION TITLE
想用该框架来做音视频通话，请问是否合适？可以给一些思路和建议吗？感谢

### DISCUSSION THREAD

**Discussion by ManyFish** [netease-im/NIM_Duilib_Framework#480] [General]
想用该框架来做音视频通话，请问是否合适？可以给一些思路和建议吗？感谢！！

**Comment by ManyFish** [comment]
视频界面的ui该使用什么控件来实现

**Comment by nmgwddj** [comment]
参考：https://github.com/netease-kit/NIM_PC_Demo/blob/master/tool_kits/ui_component/av_kit/gui/video/helper/ui_bitmap_control.cpp
